CONTRIBUTING.md

```diff
@@ -2,13 +2,33 @@
 
 ## Table of Contents
 - [Claiming an Issue](#Claiming-an-Issue)
+- [step 1. create an issue](#step-1.-create-an-issue)
+- [step 2 fork the code](#step-2-fork-the-code)
+- [step 3 Make the necessary changes](#step-3-Make-the-necessary-changes)
+- [step 4 Submit a PR](#step-4-Submit-a-PR)
+- [CONGRADUALTIONS](#CONGRADUALTIONS)
 
-## Claiming an Issue [Back to Top](#Table-of-Contents)
-
+# Claiming an Issue [Back to Top](#Table-of-Contents)
 Here are the steps to claiming an issue:
 
-1. Enter a comment that says something to the effect of "I will work on this" 
-2. wait for a 👍🏾 from one of the organizers.
-3. Fork the code in your own repo.
-4. Make the changes
-5. Submit a PR
+## step 1. create an issue [^Back to Top](#Table-of-Contents)
+
+Start with an issue # Enter a comment that says something to the effect of "I will work on this" and wait for the  one of the organizers to assigned you the issue.
+```
+#8 I'll take this issue off your hands
```

Comment:
Can you expand on this issue perhaps use the suggestion I provided

```suggestion
## Problem: Black suggests using double-qoutes. This also creates a style-guide standard of the strings should be in the `f"string {value['attr']}"` format.

Using Black to format will reduce the amount of work done solely on formatting code

### To Recreate the Problem
This is  addition so no recreated needed

### Suggested Solution
Implement Black in either CI/CD or in pre-commit

here is the code for black in pre-commit
\```
  - repo: https://github.com/psf/black
    rev: 22.10.0
    hooks:
      - id: black
        name: Format Python
        language_version: python3.10
\```

## Would you like to work on this change?
Yes 😄 
```